scala/scala-impl/src/org/jetbrains/plugins/scala/project/template/Downloader.scala

```diff
@@ -73,8 +74,8 @@ object Downloader {
 
   private def sbtCommandsForHydra(platform: Platform, version: String) = Seq(
     s"""set scalaVersion := "${version.split("_")(0)}"""",
-    s"""set credentials := Seq(Credentials("Artifactory Realm", "repo.triplequote.com", "${HydraCredentialsManager.getLogin}", "${HydraCredentialsManager.getPlainPassword}"))""",
-    s"""set resolvers := Seq(Resolver.url("Triplequote Plugins Ivy Releases", url("https://repo.triplequote.com/artifactory/ivy-releases/"))(Resolver.ivyStylePatterns), Resolver.url("Triplequote sbt-plugin-relseases", url("https://repo.triplequote.com/artifactory/sbt-plugins-release/"))(Resolver.ivyStylePatterns),  "Triplequote Plugins Releases" at "https://repo.triplequote.com/artifactory/libs-release-local/")""",
+    s"""set credentials := Seq(Credentials("Artifactory Realm", "${HydraRepository.getRepositoryName}", "${HydraCredentialsManager.getLogin}", "${HydraCredentialsManager.getPlainPassword}"))""",
```

Comment:
Aha, the fact that here we use `Artifactory Realm` is actually a problem, since the realm name depends on the specific setup. I'm not sure what we should do here...